docs/topics/native/native-improving-compilation-time.md

```diff
@@ -153,7 +153,39 @@ This feature is [Experimental](components-stability.md#stability-levels-explaine
 add the `kotlin.incremental.native=true` option to your `gradle.properties` file. If you face any problems,
 create an [issue in YouTrack](https://kotl.in/issue).
 
+### Customize LLVM optimization passes
+
+The default behavior for an `-opt` compilation is to run the same LLVM optimization passes that clang would for C++.
```

Comment:
The rest of the document uses "a release binary" instead of "an `-opt` compilation". Let's use uniform wording to make sure everyone understands that it is the same thing.